bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableGrpcClientOptions.java

```diff
@@ -22,16 +22,25 @@
  */
 public class BigtableGrpcClientOptions {
 
+  private static int DEFAULT_STREAMING_BUFFER_SIZE = BigtableGrpcClient.SCANNER_BUFFER_SIZE;
+  // We can timeout when reading large cells with a low value here. With a 10MB
+  // cell limit, 60 seconds allows our connection to drop to ~170kbyte/s. A 10 second
+  // timeout requires 1Mbyte/s
+  private static int DEFAULT_READ_PARTIAL_ROW_TIMEOUT_MILLIS =
```

Comment:
60 * 1000? I remember we were doing something like this in other PRs. Also MILLIS -> MS